target/linux/qualcommax/ipq807x/base-files/etc/hotplug.d/firmware/11-ath11k-caldata

```diff
@@ -27,15 +28,15 @@ case "$FIRMWARE" in
 		caldata_extract "0:art" 0x1000 0x20000
 		;;
 	prpl,haze)
-                caldata_extract_mmc "0:ART" 0x1000 0x20000
```

Comment:
I guess this change is unintentionally?

Reply:
This is just an indent correction.

Reply:
All of these need to go into a separate commit.